.github/workflows/ci.yml

```diff
@@ -0,0 +1,94 @@
+name: CI
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+  push:
+    branches:
+      - master
+env:
+  IMAGE_TAG: ${{ github.sha }}
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/cache@v1
+        with:
+          path: node_modules
+          key: ${{ runner.os }}-yarn-client-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-client-
+      - name: yarn
+        run: yarn
+      - name: Lint
+        run: yarn lint
+      - name: Unit Test
+        run: yarn test
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/cache@v1
+        with:
+          path: node_modules
+          key: ${{ runner.os }}-yarn-client-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-client-
+      - name: yarn
+        run: yarn
+      - name: Build
+        run: make build
+      - name: Save Docker images
+        run: |
+          mkdir -p docker-image
+          docker save -o docker-image/reviewer-client.tar libero/reviewer-client:${IMAGE_TAG}
+          gzip docker-image/reviewer-client.tar
+      - name: Upload prod Docker image
+        uses: actions/upload-artifact@v1
+        with:
+          name: docker-image
+          path: docker-image
+  browser-test:
+    needs: [test, build]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/cache@v1
+        with:
+          path: node_modules
+          key: ${{ runner.os }}-yarn-client-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-client-
+      - name: yarn
+        run: yarn
+      - name: Download Docker image artifact
+        uses: actions/download-artifact@v1
+        with:
+          name: docker-image
+      - name: Load Docker image
+        run: |
+          gunzip docker-image/reviewer-client.tar.gz
+          docker load -i docker-image/reviewer-client.tar
+      - name: Create Config
+        run: make setup
+      - name: Browser Test
+        run: make test_browser
```

Comment:
the browser tests seem to run from the host towards the application container, but magically they consistently use a consistent version between me and Github Actions (`Using: chrome (78.0.3904.108) on Linux platform`); is this managed by Nightwatch or is it luck?

Reply:
@will-byrne will know more about this then myself but I believe this is nightwatch's doing. We're hoping to integrate browserstack instead of rely on the chrome only test soon

Reply:
It will (as far as i can tell) use the local version of chrome, I think we just all have up to date versions. If it is managed by Nightwatch that is a surprise and news to me as its not pulling down a chrome binary.

Reply:
This in my opinion is the point of the build that could bite the most; I'm used to Selenium tests requiring to pin the version of the browser to get a chance at a consistent result across time; that would require Chrome (and/or the tests themselves?) to come from a container so that when it fails in CI, it fails also locally and on everyone's laptop.
You have the best information about stability here so you make the call.

Reply:
Have raised https://github.com/libero/reviewer-client/issues/64 which should address this. I think this is adding value so we can make more stable as a future enhancement.